manifests/t/TeamProwlarr/Prowlarr/1.10.5.4116/TeamProwlarr.Prowlarr.installer.yaml

```
@@ -0,0 +1,24 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: TeamProwlarr.Prowlarr
+PackageVersion: 1.10.5.4116

```

```suggestion
PackageVersion: 1.10.5
```
This should match what is written to control panel